lib/expression/v2.ex

```diff
@@ -76,6 +76,19 @@ defmodule Expression.V2 do
     end
   end
 
+  @spec escape(String.t()) :: String.t()
+  def escape(expression) when is_binary(expression) do
+    String.replace(expression, ~r/@([a-z]+)(\(|\.)/i, "@@\\g{1}\\g{2}")
+  end
```

Comment:
This is just copied from the original Expressions module. Copying here rather than defdelegating as it's tiny and it'll make search & replacing `Expressions.V2` easier when we want to start doing that.